README.md

```diff
@@ -100,28 +100,64 @@ If the `tagMapping` argument is not defined, only the `prefix` tag will be added
 
 If you do not want to include a component of `prefix` as a tag, use the special name `SKIP` in `tagMapping` for that position.
 
-## Metrics
+## Profilers
 
-`statsd-jvm-profiler` will profile the following:
+`statsd-jvm-profiler` offers 3 profilers: `MemoryProfiler`, `CPUProfiler` and `JVMCPUProfiler`.
+
+The metrics for all these profilers will prefixed with the value from the `prefix` argument or it's default value: `statsd-jvm-profiler`.
+
+You can enable specific profilers through the `profilers` argument like so:
+1. Memory metrics only: `profilers=MemoryProfiler`
+2. CPU Tracing metrics only: `profilers=CPUProfiler`
```

Comment:
I'll probably rename this to something like `CPUTracingProfiler` so the distinction between it and `JVMCPUProfiler` is more clear.